entities/player/Player.tscn

```diff
@@ -130,7 +130,7 @@ script = ExtResource("1_f30xf")
 position = Vector2(34, 24)
 scale = Vector2(2, 2)
 sprite_frames = SubResource("SpriteFrames_0nkkh")
-animation = &"move_down"
+animation = &"idle"
```

Comment:
Not sure how this was changed but the player movement still is the same

Reply:
I think default is move_down in master whereas it's idle in here maybe. This seems proper so.